src/desktop/view/canvasview.cpp

```diff
@@ -19,6 +21,8 @@
 
 namespace view {
 
+constexpr int rulerWidth = 18;
```

Comment:
Nit: should probably be `static`. I dunno if it's still relevant, but compilers used to be very unsmart about internal linkage, but behaved properly if given static linkage.

Reply:
I'm not aware of any modern compilers that mess this up. constexpr implies static here by the spec.